newsFetcher.js

```diff
@@ -0,0 +1,74 @@
+class NewsFetcher {
+  
+
+  getNews(callback){
+    console.log('called')
+    fetch('https://content.guardianapis.com/search?page=1&api-key=test&show-fields=body,headline,thumbnail')
+    .then(res => res.json())
+    .then(callback);
+  };
+  
+  displayNewsOnPage(data){
+ 
+    data.response.results.forEach(result => {
+    
+      const article = document.createElement('div');
```

Comment:
I like the way you split up creating these elements into different functions and then called them together -- your code looks really clean and adjustable.